packages/newspringchurchapp/src/prayer/PrayerTab/index.js

```diff
@@ -2,16 +2,14 @@ import React from 'react';
 import { View } from 'react-native';
 import { withNavigation } from 'react-navigation';
 import PropTypes from 'prop-types';
-import {
-  PaddedView,
-  styled,
-  Button,
-  BodyText,
-  H4,
-} from '@apollosproject/ui-kit';
+import { styled, Button, BodyText, H4 } from '@apollosproject/ui-kit';
 
 const VerticalPaddedView = styled(({ theme }) => ({
-  paddingBottom: theme.sizing.baseUnit,
+  paddingVertical: theme.sizing.baseUnit * 2,
```

Comment:
```suggestion
  paddingVertical: theme.sizing.baseUnit,
```
This can actually just stay at `baseUnit`. On the smaller Android phone the "Start Praying" button goes under the tab bar with it set at "* 2".

![Screenshot 2019-07-15 14 10 30](https://user-images.githubusercontent.com/3229463/61238547-d9607680-a70a-11e9-9467-b51813e324b8.png)

Reply:
👍